ensemble/volren/tests/test_volume_renderer.py

```diff
@@ -73,6 +74,18 @@ def _example_volume_mask(self, volume_array):
         mask[axis_slices[0], axis_slices[1], axis_slices[2]] = 255
         return mask
 
+    def _get_sample_filter(self):
+        """
+            Returns an instance of a sample filter
+        """
+        class SampleFilter(VolumeFilter):
+            name = "Sample"
+
+            def filter(self, raw_data):
+                return np.rot90(raw_data, 2)
+
+        return SampleFilter()
```

Comment:
It's unusual to build classes on the fly in methods. Please define the `SampleFilter` class at the start of the file, and create an instance instead of calling `_get_sample_filter`